### PR TITLE
Add IMU noise + basic EKF configuration

### PR DESCRIPTION
**Pull request by rafal-gorecki**
Should be merged after: #225

### Modifications

- Fix EKF
- Add IMU noise
- Rearrange urdf for Gazebo sim
- Adjust covariance values

Info: Before official alpha release it's important to adjust covariance values on real platform.

### CODE REVIEW

File: panther_bringup/config/ekf.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /**:
   ekf_node:
+    # Ref: http://docs.ros.org/en/noetic/api/robot_localization/html/state_estimation_nodes.html
     ros__parameters:
-      frequency: 50.0
+      frequency: 30.0
```
Comment (rafal-gorecki) [inline panther_bringup/config/ekf.yaml]:
```suggestion
      frequency: 50.0
```

File: panther_bringup/config/ekf.yaml [hunk 2]
```diff
@@ -15,30 +16,34 @@
...
       imu0_relative: true
-      imu0_queue_size: 5
-
+      imu0_queue_size: 3
       imu0_remove_gravitational_acceleration: false
 
+      use_control: true
+      control_timeout: 0.5
+      control_config: [true, true, false, false, false, true]
+      acceleration_limits: [2.7, 1.5, 0.0, 0.0, 0.0, 5.7]
```
Comment (rafal-gorecki) [inline panther_bringup/config/ekf.yaml]:
```suggestion
      acceleration_limits: [2.7, 1.5, 0.0, 0.0, 0.0, 5.7] # Values taken from WH01_controller.yaml and WH02_controller.yaml inside panther_controller/config
```

File: panther_description/urdf/gazebo.urdf.xacro
Comment (pawelirh) [inline panther_description/urdf/gazebo.urdf.xacro:1]: Please apply formatting to this file

File: panther_description/urdf/panther_macro.urdf.xacro
Comment (pawelirh) [inline panther_description/urdf/panther_macro.urdf.xacro:1]: Same as above, if u could format this file after changes

**Review by pawelirh — CHANGES_REQUESTED** [review]

**Review by pawelirh — APPROVED** [review]
OK 👍🏼

**Pull request merged**